File: listas.js
// Creamos una lista (array) en JavaScript
let a = [1, 2, 3, 4];
console.log(a);

// Operamos como si fuera un "basestring" (accedemos a un valor de índice)
console.log(a[0]);

// Imprimimos un rango de elementos en el array
console.log(a.slice(0, 2));

// Concatenamos la lista con otra lista o un número
let b = [5, 6];
console.log(a.concat(b));

// Podemos sustituir el valor de una posición
a[3] = 'Hola';
console.log(a);

// Usamos el método push() para añadir un elemento (equivalente a append() en Python)
a.push(3);
console.log(a);

// Al igual que en los strings, podemos obtener el tamaño con .length
console.log(a.length);

// Comprobamos si un elemento está en el array
console.log(a.includes(5)); // Retorna true si 5 está en el array
console.log(a.includes(1)); // Retorna true si 1 está en el array

File: Python/Listas.py
a = [1,2,3,4]
print(a)

#Operaremos como si fuese un basestring

print(a[0])

print(a[0:2])

#Concatenaremos lista con un otra lista o un número

b = [5,6]

print(a + b)

#Podemos sustituir el valor de una posición

a[3] = 'Hola'
print(a)

#Podemos usar el método append() para añadir un elemento

a.append(3)

print(a)

#Al igual que en los strings, podemos usar el método len()

print(len(a))

#Podemos ver si un elemento está o no en una lista

print(5 in a)

print(1 in a)